recipes/result/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (C) 2015-2020 Danilo Bargen and contributors
```

Comment:
Already included in the PyPI tarball used here. Please remove.

Reply:
Great, wasn't aware of this. Removed it!